nsds_lab_to_nwb/metadata/exp_note_reader.py

```diff
@@ -0,0 +1,160 @@
+import pandas as pd
```

Comment:
@jihyunbak Added exp_note_reader.py and changed yaml to write file using block_folder

Reply:
Thanks @jthermiz !